components/offres/RechercheOffresEmploiSecondary.tsx

```diff
@@ -8,13 +8,12 @@ import {
   SearchOffresEmploiQuery,
   TypeContrat,
 } from 'services/offres-emploi.service'
+import { FormValues } from 'types/form'
 
 type RechercheOffresEmploiSecondaryProps = {
```

Comment:
idem pour le nommage en français pour éviter confusion avec UI
```suggestion
type RechercheOffresEmploiSecondaireProps = {
type RechercheOffresEmploiAvecCriteresProps = {

```